integration/test/agent_util.go

```diff
@@ -77,11 +77,11 @@ func ReadAgentOutput(d time.Duration) string {
 	return string(out)
 }
 
-func RunShellScript(path string, args ...string) {
+func RunShellScript(path string, args ...string) error{
```

Comment:
I'm pretty sure there are other places that use this function. Please verify that the integration tests still compile

Reply:
Yes. I triggered an integration test on my fork

Reply:
Here is the result for my integration test on my fork https://github.com/khanhntd/amazon-cloudwatch-agent/actions/runs/2956867720